Source/svg/SVGParser.swift

```diff
@@ -1697,3 +1706,12 @@ fileprivate extension String {
         return String(self[start..<end])
     }
 }
+
+fileprivate class UserSpacePath: Path {
```

Comment:
It's a good practice to avoid inheritance when it's not necessary, just use composing:
```
class UserSpacePath {
    let userSpace: Bool
    let path: Path
    ...
}
```